src/CPPOverload.h

```diff
@@ -40,13 +40,15 @@ class CPPOverload {
     typedef std::vector<PyCallable*> Methods_t;
 
     struct MethodInfo_t {
-        MethodInfo_t() : fFlags(CallContext::kNone) { fRefCount = new int(1); }
+        MethodInfo_t() : fFlags(CallContext::kNone), fDoc(nullptr)
+            { fRefCount = new int(1); }
         ~MethodInfo_t();
 
         std::string                 fName;
         CPPOverload::DispatchMap_t  fDispatchMap;
         CPPOverload::Methods_t      fMethods;
         uint32_t                    fFlags;
+        PyObject*                   fDoc;
```

Comment:
Recommend to place `fDoc` above `fFlags`. The latter being half a word size on 64b platforms, so placing the smallest data members last may, in some cases, save on alignment.